config.toml

```diff
@@ -41,7 +42,7 @@ section = [
 [params]
 contact = "#"
 copyright = "<!--Creative Commons License-->This site is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/).<!--/Creative Commons License-->"
-description = "DEVSECOPS TALKS - SURVIVING CLOUD NATIVE"
+description = "DEVSECOPS TALKS - SURVIVING CLOUD NATIVE with Andrey, Julien and Mattias"
```

Comment:
The snob in me: should be in a different commit because of not matching commits/PR title and the change
The pragmatist in me: whatever
Just voicing an opinion, it is up to you to decide how to handle it

Reply:
I agree it's better to have different commits.
Although since this is in the same file and we don't need to review history since we are always on latest, I don't think it makes a lot of sense to be extreme about git best practices.